pages/productListingPage/productListingPage-vert.css

```diff
@@ -0,0 +1,95 @@
+@import "/styles.css";
+
+:root {}
+
+body {}
+
```

Comment:
can be removed if not being used.

Reply:
This is a placeholder that will be used in react app